Src/AutoFixtureUnitTest/Kernel/TerminatingWithPathSpecimenBuilderTest.cs

```diff
@@ -295,5 +317,82 @@ public void CreateOnMultipleThreadsConcurrentlyWorks()
             // Teardown
         }
 
+        [Fact]
+        public void ObjectCreationExceptionIsThrownIfCreationFails()
+        {
+            //arrange
+            var request = new object();
+            var builder = new DelegatingSpecimenBuilder
+            {
+                OnCreate = (r, c) => throw new Exception()
+            };
+
+            var sut = new TerminatingWithPathSpecimenBuilder(builder);
+            var context = new DelegatingSpecimenContext();
+
+            //act & assert
```

Comment:
Throughout the AutoFixture codebase, the [Four Phase Test](http://xunitpatterns.com/Four%20Phase%20Test.html) naming convention is used. Particularly, for existing tests, I don't see any reason to switch to [Arrange-Act-Assert](http://wiki.c2.com/?ArrangeActAssert)? 😃